pyproject.toml

```diff
@@ -123,6 +123,9 @@ select = [
     "ISC", # flake8-implicit-str-concat
     "N", # pep8-naming
     "NPY", # modern numpy
+    "PLC", # pylint
+    "PLE", # pylint
+    "PLW", # pylint
```

Comment:
Could we make the comments more specific if possible? Which format check from pylint?

Reply:
I updated to "PL" which includes all checks from pylint.